modules/references/client/components/ListReferences.component.js

```diff
@@ -82,15 +109,42 @@ export default function ListReferences({ profile, authenticatedUser }) {
     </section>
   );
 
+  const renderExperiencePairsSection = (sectionTitle, experiencePairs) => (
+    <section>
+      <div className="row">
+        <div className="col-xs-12 col-sm-6">
+          <h4 className="text-muted">{sectionTitle}</h4>
+        </div>
+      </div>
+
+      {experiencePairs.map(experiencePair => (
+        <div key={experiencePair.theirs._id}>
+          <div className="row">
+            <div className="col-xs-12">
+              <Reference reference={experiencePair.theirs} isMine={true} />
+            </div>
+          </div>
+          {experiencePair.mine && (
```

Comment:
`mine` term confuses me a bit: I could be person A looking at experiences between B and C, right?

Reply:
@simison How do you like `writtenByUser`/`sharedWithUser`?

Reply:
`incoming`/`outgoing`?

Reply:
Further ideas: `written`/`received`, `from`/`to`

Too late in the evening to form an opinion. 😆

Reply:
Just for inspiration I checked how CS does it (in my profile json dump):

<img width="496" alt="Screenshot 2020-12-06 at 23 17 07" src="https://user-images.githubusercontent.com/87168/101292614-536e2f80-3819-11eb-9619-9ca850640dd6.png">